### PR TITLE
[PBA 2657] Sync localization with skin-config

### PR DESCRIPTION
**Pull request by brol1dev**
I refactored the code a bit, so that OOSkinViewController doesn't have a lot of methods.

The main problem was that the code wasn't updated to the newer changes in skin-config, so localization wasn't working.

I wanted to add unit test for 'utils.js', but I wasn't able to setup that test case with Jest. I'm getting errors, it seems that it isn't mocking some react stuff correctly, as we require React in utils.js.

### CODE REVIEW

File: sdk/OoyalaSkinSDK/OoyalaSkinSDK/Utils/NSDictionary+Utils.m [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+//  Copyright © 2015 ooyala. All rights reserved.
+//
+
+#import "NSDictionary+Utils.h"
+
+#import <OoyalaSDK/OODebugMode.h>
+#import "OOLocaleHelper.h"
+#import "NSMutableDictionary+Utils.h"
+
+static NSString * const kLocalizableStrings = @"localization";
```
Comment (lental) [inline sdk/OoyalaSkinSDK/OoyalaSkinSDK/Utils/NSDictionary+Utils.m]: can these constants be removed from the OoSkinViewController? Are they there as well?

Comment (brol1dev) [inline sdk/OoyalaSkinSDK/OoyalaSkinSDK/Utils/NSDictionary+Utils.m]: Oh yea, let me remove them.

**Comment by brol1dev** [comment]
Oh wait @lental, I removed the constants, but didn't check before pushing. The constants are still being used in OOSkinViewController. I guess I'll keep them there and use them from the other class I created.

**Comment by brol1dev** [comment]
It's ready now.

But I think we should find where each of our constants should reside.

**Comment by lental** [comment]
I agree, not suuper happy about having unnecessary things like that in our header, but it's okay for now.

I think our locale helper should just take the skin-config variable, and localehelper itself should have the constants

**Comment by lental** [comment]
actually, do you think you can fix this?  modifying OOLocaleHelper localizedString to  accept the skinconfig, instead of skinconfig[klocalizablestrings]

**Comment by brol1dev** [comment]
Ok, will do.

**Comment by brol1dev** [comment]
Missed that one. OOLocaleHelper has the constants now.

**Comment by lental** [comment]
perfect! thanks

**Pull request merged**